packages/jest-editor-support/src/Settings.js

```diff
@@ -73,16 +70,18 @@ module.exports = class Settings extends EventEmitter {
       // See https://github.com/facebook/jest/issues/2343 for moving this into
       // the config object
       if (string.includes('jest version =')) {
-        const version = string.split('jest version =')
+        const version = string
+          .split('jest version =')
           .pop()
-          .split(EOL)[0]
-          .trim();
+          .split(EOL)
+          [0].trim();
```

Comment:
https://github.com/prettier/prettier/issues/775

If you are curious, [0] and .trim are the same ast node (MemberExpression), there are test frameworks that like to do 

```
expect(x)
  .to.not.be.a.string()
  .and.to.be.a.number();
```

so we merge all those together as a single block. It turns out that `[0]` doesn't follow this behavior and should be grouped with the previous one.